zSystems Update (#435)
SUSE/doc-susemanager: modules/installation/pages/install-ibmz.adoc

|===
| Hardware     | Recommended
| CPU          | Minimum 4 dedicated 64-bit CPU cores
| RAM:         | Test Server: Minimum 3{nbsp}GB RAM and 2{nbsp}GB Swap space
|              | Base Installation: Minimum 16{nbsp}GB
|              | Production Server: Minimum 32{nbsp}GB
| Disk Space:  | Root Partition: Minimum 100{nbsp}GB
|              | [path]``/var/lib/pgsql``: Minimum 50{nbsp}GB
|              | [path]``/var/spacewalk``: Minimum 50{nbsp}GB per SUSE product and 250{nbsp}GB per Red Hat product
|===

[IMPORTANT]
====
Memory should be split across available RAM, VDISK, and swap to suit your environment.
On a production system the ratio of physical memory to VDISK will need to be evaluated based on the number of clients you will be installing.
====

You will require an additional disk for database storage.
This should be an [systemitem]``zFCP`` or [systemitem]``DASD`` device as these are preferred for use with [systemitem]``HYPERPAV``.
The database storage disk should have:

* At least 50{nbsp}GB for [path]``/var/lib/pgsql``
* At least 50{nbsp}GB for each SUSE product in [path]``/var/spacewalk``
* At least 250{nbsp}GB for each Red Hat product in [path]``/var/spacewalk``

> You will need to ensure you have sufficient disk storage for {productname}

Reviewer: ```suggestion
You will need to ensure you have sufficient disk storage for {productname} before running [command]``yast2 susemanagersetup``.
```
avoid line break

> before running [command]``yast2 susemanagersetup``.

Reviewer: ```suggestion
```
cont. of the previous suggestion

By default, the {productname} file system, including the embedded database and patch directories, reside within the root directory.

> While adjustments are possible once installation is complete, it is important that specify and monitor these adjustments closely.

Reviewer: ```suggestion
While adjustments are possible when installation is complete, it is important that you specify and monitor these adjustments closely.
```
simplify wording

> For information on storage management and reclaiming disk space, see the {productname} Administration Guide.

Reviewer: is it possible to provide a clickable link?  Wondering whether that's actually in our Admin Guide.  Maybe, it's in the SLES Admin?

Author: Oh yeah, I meant to come back to that. I will need to investigate.

Author: I can't see anything specific in our Admin Guide, but I think it's a good candidate for a troubleshooting  section. I'll create something in a different PR.

Author: https://github.com/SUSE/doc-susemanager/pull/437 (it's a beginning)

[WARNING]
====
If your {productname} runs out of disk space, this can have a severe impact on its database and file structure.
A full recovery is only possible with a previous backup or a new {productname} installation.
{suse} technical services will not be able to provide support for systems suffering from low disk space conditions.
====

.Network Requirements:

* OSA Express Ethernet (including Fast and Gigabit Ethernet)
* HiperSockets or Guest LAN
* 10{nbsp}GBE, VSWITCH
* RDMA over Converged Ethernet (RoCE)

These interfaces are still included but no longer supported:

* CTC or virtual CTC
* IP network interface for IUCV

> The z/VM guest you want to run {productname} from will require a static IP address and hostname before you begin, as these can not easily be changed after initial installation.

Reviewer: ```suggestion
The z/VM guest you want to run {productname} from will require a static IP address and hostname before you begin, as these cannot easily be changed after initial installation.
```
normalization
